indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -157,7 +165,18 @@ function EditableListDisplay({
         options: [
           {value: 'not_submitted', text: Translate.string('Not submitted'), color: 'default'},
           {value: 'ready_for_review', text: Translate.string('Ready for review'), color: 'grey'},
-          {value: 'accepted', text: Translate.string('Accepted'), exclusive: true, color: 'green'},
+          {
+            value: 'accepted',
+            text: Translate.string('Accepted by editor'),
```

Comment:
```suggestion
            text: Translate.string('Accepted'),
```

No super strong feelings on this one, but I'd maybe keep the same text here as before. That table looks a bit cluttered already as is, so the simpler the better